relengapi/blueprints/slaveloan/bugzilla.py

```diff
@@ -0,0 +1,142 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# XXX Much of this functionality should probably be its own
+#     Separate relengapi blueprint
+
+import logging
+
+from bzrest.client import BugzillaClient
+from flask import current_app
+from flask import url_for
+
+log = logging.getLogger(__name__)
+
+MAX_ALIAS = 15
+DEFAULT_BUGZILLA_URL = "https://bugzilla-dev.allizom.org/rest/"
+
+
+def _bzclient():
+    if getattr(current_app, 'bzclient', None):
+        return current_app.bzclient
+    # XXX: Todo log warning if user or pass is "None"
+    bzclient = BugzillaClient()
+    bzclient.configure(bzurl=current_app.config.get('BUGZILLA_URL', DEFAULT_BUGZILLA_URL),
+                       username=current_app.config.get('BUGZILLA_USER', None),
+                       password=current_app.config.get('BUGZILLA_PASS', None))
+    current_app.bzclient = bzclient
+    return current_app.bzclient
+
+
+class Bug(object):
+    alias = None
+    id = None
+
+    def __init__(self, id_=None, loadInfo=True):
```

Comment:
What does it mean to pass `None` as `id_` here?  It doesn't look like anything would ever set it in that case.